src/virtualenv/seed/via_app_data/pip_install/base.py

```diff
@@ -157,6 +157,8 @@ def _write_script(scriptnames, shebang, script, filenames, ext):
                 "{}{}".format(name, version_info.major),
                 "{}-{}.{}".format(name, version_info.major, version_info.minor),
```

Comment:
Why are we generating "scriptname-X.Y" scripts?

Reply:
Because that's what distlib does, and the idea is that the virtualenv should shield you from calling os versions of these 👍